apps/frontend/src/app/3_organisms/ZeroLocForm/constants.ts

```diff
@@ -1,9 +1,10 @@
 import { SupportedTokens } from '@sovryn/contracts';
+import { Decimal } from '@sovryn/utils';
 
-export const CRITICAL_COLLATERAL_RATIO = 1.5; // 150%
-export const MINIMUM_COLLATERAL_RATIO = 1.1; // 110%
+export const CRITICAL_COLLATERAL_RATIO = Decimal.from(1.5); // 150%
+export const MINIMUM_COLLATERAL_RATIO = Decimal.from(1.1); // 110%
 
-export const MIN_DEBT_SIZE = 200; // 200 ZUSD
+export const MIN_DEBT_SIZE = Decimal.from(200); // 200 ZUSD
```

Comment:
I added a couple of comments elsewhere indicating we should use `LIQUIDATION_RESERVE_AMOUNT` from `app/src/utils/constants.ts`, but if we do that, we should probably refactor it to here as well since it is used alongside the rest of these constants

Reply:
I see LIQUIDATION_RESERVE_AMOUNT also used in histories so leaving constant in global constants, so this module would not be downloaded if user does not browse to the page